tests/func-tests/console_plugin_test.go

```diff
@@ -65,29 +65,35 @@ var _ = Describe("kubevirt console plugin", Label(tests.OpenshiftLabel, "console
 		pluginServiceName := kubevirtPlugin.Spec.Backend.Service.Name
 		pluginServicePort := kubevirtPlugin.Spec.Backend.Service.Port
 
-		consolePods := &corev1.PodList{}
-		Expect(cli.List(ctx, consolePods, client.MatchingLabels{
-			"app":       "console",
-			"component": "ui",
-		}, client.InNamespace(openshiftConsoleNamespace))).To(Succeed())
-
-		Expect(consolePods.Items).ToNot(BeEmpty())
-
-		testConsolePod := consolePods.Items[0]
-		command := fmt.Sprintf(`curl -ks https://%s.%s.svc:%d/plugin-manifest.json`,
-			pluginServiceName, tests.InstallNamespace, pluginServicePort)
-
-		stdout, stderr, err := executeCommandOnPod(ctx, k8sClientSet, &testConsolePod, command)
-		Expect(err).ToNot(HaveOccurred())
-		Expect(stdout).ToNot(BeEmpty())
-		Expect(stderr).To(BeEmpty())
-
-		var pluginManifests map[string]interface{}
-		err = json.Unmarshal([]byte(stdout), &pluginManifests)
-		Expect(err).ToNot(HaveOccurred())
-
-		pluginName := pluginManifests["name"]
-		Expect(pluginName).To(Equal(expectedKubevirtConsolePluginName))
+		hcoPods := &corev1.PodList{}
+		Expect(cli.List(ctx, hcoPods, client.MatchingLabels{
+			"name": "hyperconverged-cluster-operator",
+		}, client.InNamespace(tests.InstallNamespace))).To(Succeed())
+
+		Expect(hcoPods.Items).ToNot(BeEmpty())
+
+		testConsolePod := hcoPods.Items[0]
+		command := fmt.Sprintf(`curl -ks https://%s:%d/plugin-manifest.json`,
+			pluginServiceName, pluginServicePort)
+
+		Eventually(func(g Gomega, ctx context.Context) string {
```

Comment:
maybe let's add here a comment with the console bug number? that this Eventually can be removed once it's fixed.

Reply:
I prefer to not add comments